generators/client-2/templates/src/main/webapp/app/components/alert/_alert-error.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, Inject, OnDestroy } from '@angular/core';
+import { AlertService } from "./alert.service";
+
+@Component({
+    selector: 'jhi-alert-error',
+    template: '<div class="alerts" ng-cloak="">' +
```

Comment:
we can use es6 string literals to avoid the concatenation. I'll merge and update this